### ISSUE TITLE
GetItemInfo Error

### ISSUE THREAD

**Issue by gaveer** [GoldpawsStuff/Bagnon_BoE#21]
```lua
10x Bagnon_BoE/main.lua:94: Usage: GetItemInfo(itemID|"name"|"itemlink")
[string "=[C]"]: ?
[string "@Bagnon_BoE/main.lua"]:94: in function `update'
[string "@Bagnon_BoE/dispatcher.lua"]:76: in function <Bagnon_BoE/dispatcher.lua:74>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "@BagBrother/libs/Poncho-2.0-3/Poncho-2.0.lua"]:40: in function `Update'
[string "@BagBrother/frames/containers/item.lua"]:72: in function <BagBrother/frames/containers/item.lua:71>
[string "=[C]"]: in function `Show'
[string "@BagBrother/core/classes/item.lua"]:47: in function <BagBrother/core/classes/item.lua:39>
[string "=(tail call)"]: ?
[string "@BagBrother/core/classes/itemGroup.lua"]:102: in function `Layout'
[string "@BagBrother/core/classes/itemGroup.lua"]:32: in function `Update'
[string "@BagBrother/core/classes/itemGroup.lua"]:26: in function <BagBrother/core/classes/itemGroup.lua:12>
[string "=(tail call)"]: ?
[string "@Bagnon/src/frame.lua"]:24: in function `New'
[string "@BagBrother/core/api/frames.lua"]:83: in function `New'
[string "@BagBrother/core/api/frames.lua"]:28: in function `Show'
[string "@BagBrother/core/api/frames.lua"]:24: in function <BagBrother/core/api/frames.lua:23>
[string "=[C]"]: in function `Toggle'
[string "@BagBrother/core/classes/offlineSelector.lua"]:91: in function <...eBagBrother/core/classes/offlineSelector.lua:54>

Locals:
(*temporary) = nil

```

**Comment by gaveer** [comment]
@Goldpaw

**Comment by Goldpaw** [comment]
Fixed in 2.0.77-Release
